### ISSUE TITLE
Adding support for syncing non-epub books on Kobo device

### ISSUE THREAD

**Issue by NSeydoux** [janeczku/calibre-web#3095]
# Problem description

I'm using the Kobo sync feature, which is extremely convenient. However, only the books with an available epub format are synced on the device, which supports reading PDFs and CBZ documents as well. Some of the books I have are in non-epub format, at it would be convenient to support syncing them natively rather than requiring prior conversion.

# Suggested new feature

I would like all books to be synced on the kobo device, regardless of their format. 

# Alternatives considered

An alternative would be to manually convert books to epub, and have the converted books stored along the native ones.

Storing the books on the device manually via USB is also a functional workaround. This feature would be an improvement to mu user experience, but I'm not blocked by this at all, so there's no "urgency" to it.

# Additional context

N/A

**Comment by NSeydoux** [comment]
I went through the existing issues, and didn't find an issue describing this specific feature, but if I missed it I'm happy for this to be closed as a duplicate.

Moreover, I'm a developer, and I will look into the code to see how this could be implemented. Any guidance/suggestions of where I should look first are welcome, as I'm not familiar with this codebase, and python is not my primary development language.

**Comment by OzzieIsaacs** [comment]
Would be nice to have this, but the kobo is only syncing (k)epubs, no matter what you do, so you might try yourself, by chaning this line:
https://github.com/janeczku/calibre-web/blob/ded4cb7a196920e09a3a9a3eb45b69352f15909b/cps/kobo.py#L57
to e.g.
KOBO_FORMATS = {"KEPUB": ["KEPUB"], "EPUB": ["EPUB3", "EPUB"], "PDF": ["PDF"]}
The kobo reader will ignore this

**Comment by NSeydoux** [comment]
Thanks for the fast response!
